### PR TITLE
Update PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by eltociear**
## Description

CONTRIBUTIONS.md link fix.

before
* Please include a summary of the changes and be sure to follow our [Contribution Guidelines](../CONTRIBUTIONS.md).

after
* Please include a summary of the changes and be sure to follow our [Contribution Guidelines](../blob/1.x/CONTRIBUTIONS.md).

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## Description
 
-Please include a summary of the changes and be sure to follow our [Contribution Guidelines](../CONTRIBUTIONS.md).
+Please include a summary of the changes and be sure to follow our [Contribution Guidelines](../blob/1.x/CONTRIBUTIONS.md).
```
Comment (nazarhussain) [inline .github/PULL_REQUEST_TEMPLATE.md:3]: Why we need to update this link? Should not it always be pointing to default branch?

**Review by nazarhussain — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (luu-alex) [reply .github/PULL_REQUEST_TEMPLATE.md:3]: Clicking on the before contribution link above, as well as when [contributors](https://github.com/ChainSafe/web3.js/pull/4864#discussion_r830292670) make PRs leads to a [404](https://github.com/ChainSafe/web3.js/CONTRIBUTIONS.md)

Comment (eltociear) [reply .github/PULL_REQUEST_TEMPLATE.md:3]: @luu-alex @nazarhussain Thanks that's reason.

Comment (luu-alex) [reply .github/PULL_REQUEST_TEMPLATE.md:3]: I think this PR is fine, but we may need a different PR template for 4.x

**Review by luu-alex — APPROVED** [review]

**Review by jdevcs — APPROVED** [review]

**Review by nazarhussain — APPROVED** [review]

**Pull request merged**